lib/galaxy_test/base/populators.py

```diff
@@ -1195,13 +1196,7 @@ def reimport_history(self, history_id, history_name, wait_on_history_length, exp
 
         return imported_history_id
 
-    def get_random_name(self, prefix=None, suffix=None, len=10):
-        # stolen from navigates_galaxy.py
-        return "{}{}{}".format(
-            prefix or "",
-            "".join(random.choice(string.ascii_lowercase + string.digits) for _ in range(len)),
-            suffix or "",
-        )
+    get_random_name = random_name
 
```

Comment:
```suggestion
```

Reply:
This needed to be a class method - wasn't consuming self, etc... I've reworked this.